content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0.md

```diff
@@ -29,6 +29,10 @@ using com.apama.functional.Functional;
 
 The **Cron Timer** block now has a **Time Zone** parameter to customize which time zone this block is relative to.
 
+### First-time activation of Analytics Builder
+
+In some circumstances, Analytics Builder must be activated by the user before it is used for the first time. If you see the 'Activate' button for the Analytics Builder in the Home page of the Streaming Analytics application, pressing this will start the process. This process may take a few minutes.
```

Comment:
A small suggestion, I have observed documentations uses `click` action for buttons instead of `press` so '...clicking this will start the process`?

Reply:
Sounds good to me. Karin just suggested that and I've committed her suggestion.